src/css/cs/items/_kitchensink.scss

```diff
@@ -271,7 +271,7 @@
 
   // Custom font-widths, PS does not deliver classes to allow different font-sizes
   @for $i from 10 through 25 {
-    $width: $i * 1px;
+    $width: $i * 5px;
```

Comment:
No that wasn't intended.  Not sure why I don't have the margin-width showing up in my local any more?..i'm looking into it.

In the meantime I'm changing the font-width back to "1px"